src/transform/adm0_surround.py

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+# Process Surrounding Countries for GADM AOI.
+#############################################################################
+
+import sys
+import os
+import fiona
+import geopandas as gpd
+import logging
+import datetime
+import pandas as pd
+from jsonschema import validate
+
+from utils.yaml_api import parse_yaml 
+from utils import gadm_gpkg_processing
+
+import pdb
+
+##
+#
+def transform():
+    config = parse_yaml('config.yml')
+    held_gpkg = os.path.join(
+            config['dirs']['raw_data'], config['surrounding']['gadm']['raw'])
+    country_aoi = config['constants']['ISO3']
+    schema_filename = os.path.join(
+            config['dirs']['schemas'], config['surrounding']['schema'])
+    output_filename = os.path.join(
+            config['dirs']['processed_data'], 
+            config['surrounding']['gadm']['processed'])
+    print(f'Reading {held_gpkg}')
+    for layername in fiona.listlayers(held_gpkg):
```

Comment:
It seems there is only one layer in the geopackage (called "gadm"). What happens if you just read the file directdly into geopandas without specifying the layer?